docs/book/src/plugins/go-v3-plugin.md

```diff
@@ -0,0 +1,57 @@
+# go/v3
+
+Kubebuilder tool will scaffold the go/v3 plugin by default. This plugin is a composition of the plugins ` kustomize.common.kubebuilder.io/v1` and `base.go.kubebuilder.io/v3`. By using you can scaffold the default project which is a helper to construct sets of [controllers][controller-runtime]. 
+
+It basically scaffolds all the boilerplate code required to create <b>and</b> design controllers. Note that by following the [quickstart][quickstart] you will be using this plugin. 
```

Comment:
Why do we need the bold?

Reply:
Oh sorry. I was checking something and forgot to remove it. Will push the changes